op-acceptance-tests/justfile

```diff
@@ -6,13 +6,15 @@ ACCEPTOR_IMAGE := env_var_or_default("ACCEPTOR_IMAGE", DOCKER_REGISTRY + "/op-ac
 
 # Default recipe - runs acceptance tests
 default:
-    @just acceptance-test
+    @just acceptance-test interop interop
 
 # Run acceptance tests with mise-managed binary
 acceptance-test devnet="simple" gate="localnet":
```

Comment:
Feels like we should either remove the defaults or maintain template aliases for devnet/gate that always refer to the latest release/release gate.

Reply:
I went with a hybrid for now:
* default is a "simple" devnet, testing holocent
* aliases for the upcoming releases: isthmus & interop